sdk/python/test/e2e/constants.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 kubeflow.org.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+TEST_GANG_SCHEDULER_NAME_ENV_KEY = "GANG_SCHEDULER_NAME"
+TEST_GANG_SCHEDULER_NAME_SCHEDULER_PLUGINS = "scheduler-plugins"
+TEST_GANG_SCHEDULER_NAME_VOLCANO = "volcano"
+TEST_GANG_SCHEDULER_NAME_NONE = "none"
+
+GANG_SCHEDULERS = {TEST_GANG_SCHEDULER_NAME_SCHEDULER_PLUGINS, TEST_GANG_SCHEDULER_NAME_VOLCANO}
+NONE_GANG_SCHEDULERS = {TEST_GANG_SCHEDULER_NAME_NONE, ""}
```

Comment:
Since we send only `none` scheduler via our tests, do we need "" in this dict ?

Reply:
As you say, we always set a `GANG_SCHEDULER_NAME ` env var in our gh-actions. 
Although, I would like to run E2E for plain scheduling locally without setting a `GANG_SCHEDULER_NAME ` env var.

It might be helpful to run E2E locally.

@andreyvelich WDYT?

Reply:
Sure, sound good!